clusterloader/README.md

```diff
@@ -0,0 +1,76 @@
+# Cluster Loader
+
+## Building
+In order to vendor in e2e/framework & deps we have used [glide](https://github.com/Masterminds/glide).
+After installing glide just run:
+```
+cd perf-tests/clusterloader/
+glide install -v
```

Comment:
Checked in glide.lock with versions. Now the glide install -v will just use the versions in that file.